lib/_http_incoming.js

```diff
@@ -60,25 +59,21 @@ function IncomingMessage(socket) {
 }
 util.inherits(IncomingMessage, Stream.Readable);
 
-
 exports.IncomingMessage = IncomingMessage;
 
-
 IncomingMessage.prototype.setTimeout = function(msecs, callback) {
   if (callback)
     this.on('timeout', callback);
   this.socket.setTimeout(msecs);
 };
 
-
 IncomingMessage.prototype.read = function(n) {
   this._consuming = true;
   this.read = Stream.Readable.prototype.read;
   return this.read(n);
 };
 
-
-IncomingMessage.prototype._read = function(n) {
+IncomingMessage.prototype._read = function() {
```

Comment:
There's an option for args: http://eslint.org/docs/rules/no-unused-vars.html#args, I think we could use `after-used` or even `none`.